lib/ctblgrp.gi

```diff
@@ -729,15 +705,21 @@ InstallGlobalFunction(SplitStep,function(D,bestMat)
       dim:=raum.dim;
       # cut out the 'active part' for computation of an eigenbase
       activeCols:=DxActiveCols(D,raum);
-      N:= NullMat( dim, dim, o );
+      N:= ZeroMatrix(f,dim,dim);
       for row in [1..dim] do
-        Row:=base[row]*M;
+        #Row:=base[row]*M;
+        Row:=CompatibleVector(M);
+        for col in [1..Length(Row)] do
+          Row[col]:=base[row,col];
+        od;
+        Row:=Row*M;
```

Comment:
Note to self: I think we need a function doing this; i.e. which takes a matrix `M`, and a list of co/a vector `v`, and returns a new vector in "compatible" representation. 

Though in this case, perhaps `base` could be turned into a list of vectors of the right type from the start, i.e. as part of its construction?

Reply:
@fingolfin What about the syntax `Vector( <M>, <v> )`?